### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by RDIL**
Lots of cool updates
sorry if this breaks stuff just trying to help

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -5,7 +5,9 @@ build_task:
   use_compute_credits: $CIRRUS_BRANCH == 'master'
   node_modules_cache:
     folder: node_modules
-    fingerprint_script: cat yarn.lock
+    fingerprint_script:
+      - cat yarn.lock
+      - cat package.json
```
Comment (fkorotkov) [inline .cirrus.yml]: `yarn.lock` should be enough for the fingerprint

**Comment by fkorotkov** [comment]
I have a few things to do with the UI in the next couple of days. Will merge this change once I'll start working on them to do some extra testing. Thanks!

**Comment by RDIL** [comment]
No worries, I’m going to squash all these commits together in the meantime.

**Comment by RDIL** [comment]
So I just used the normal git command in cmd for the first time ever instead of gitkracken and I'm not sure if it worked, I would suggest cherry-picking f520ebcdbadef1e6f98b5d565becaad10b94b78f when merging into master.

**Pull request merged**